common/lib/capa/capa/templates/choicegroup.html

```diff
@@ -33,8 +32,9 @@
                         <% label_class += ' choicegroup_' + correctness %>
                     % endif
                 % endif
-                class="${label_class}" >
-
+                class="${label_class}"
+                ${describedby}
```

Comment:
@muhammad-ammar this is a bit confusing as by at looking at it one can't determine what `${describedby}` is if would be like `describedby=${description_ids}` it is much more understandable.